src/components/PeopleFilters.tsx

```diff
@@ -2,17 +1,55 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
+type Props = {};
+
+export const PeopleFilters: React.FC<Props> = ({}) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const sexStatus = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries');
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(
+      getSearchWith(searchParams, { query: event.target.value || null }),
+    );
+  };
+
+  const toggleCentury = (century: string) => {
+    return centuries.includes(century)
+      ? centuries.filter(c => c !== century)
+      : [...centuries, century];
+  };
```

Comment:
The `toggleCentury` function is not very intuitive without context. It's better to name functions in a way that clearly describes what they do. In this case, you could rename it to `getUpdatedCenturies` or `updateSelectedCenturies`.